crates/ide-assists/src/handlers/add_missing_impl_members.rs

```diff
@@ -418,6 +418,60 @@ impl<'x, 'y, T, V, U: Default> Trait<'x, 'y, T, V, U> for () {
         );
     }
 
+    #[test]
+    fn test_const_substitution() {
```

Comment:
Can you add one more test where const params appear inside types? Something like:

```rust
trait Foo<const N: usize> {
    fn foo(_: [i32; N]) -> OtherType<N> { loop {} }
}
```